src/lib/components/chatMessages/ReasoningSegment.svelte

```
@@ -1,79 +1,131 @@
+<script lang="ts" module>
+  export type ReasoningSegmentState = "default" | "closed" | "open";
+  export const MAX_STREAMING_HEIGHT = 200;
+</script>
+
+<!-- svelte-ignore state_referenced_locally -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)